crates/uv/src/cli.rs

```diff
@@ -1347,7 +1348,7 @@ pub(crate) struct RunArgs {
 
     /// The arguments to the command.
     #[arg(allow_hyphen_values = true)]
-    pub(crate) args: Vec<String>,
+    pub(crate) args: Vec<OsString>,
```

Comment:
We need this to insert a properly insert a path into the arguments, but this seems generally "more correct".

We'll want to do this for `target` too.